CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 #### Assembly
 - Expanded capabilities of the `debug` decorator. Added `debug.mem` and `debug.local` variations.
 - Introduced the `emit.<event_id>` assembly instruction (#1119).
+- Introduced the `procref.<proc_name>` assembly instruction (#1113).
```

Comment:
nit: could we also add the PR number to the first line above (I missed this in the other PR review).